### ISSUE TITLE
Mobile notification page broken

### ISSUE THREAD

**Issue by Flaburgan** [diaspora/diaspora#5778]
```
Started GET "/notifications" for X.X.X.X at 2015-03-14 19:10:16 +0100
Processing by NotificationsController#index as HTML
  Rendered notifications/_notification.html.haml (18.9ms)
  Rendered notifications/_notification.html.haml (7.7ms)
  Rendered notifications/_notification.html.haml (13.6ms)
  Rendered notifications/_notification.html.haml (9.5ms)
  Rendered aspect_memberships/_aspect_membership_dropdown.mobile.haml (0.8ms)
  Rendered notifications/_notification.html.haml (9.2ms)
Completed 500 Internal Server Error in 105ms

ActionView::Template::Error (undefined method `id' for nil:NilClass):
    1: %div
    2:   %select{:name => 'user_aspects', :class => 'user_aspects', 'data-person-id' => @person.id}
    3:     %option{:value => 'list_cover', :class => 'list_cover', :disabled => 'true', :selected => 'true'}
    4:       = t("add_contact")
    5:     - contact = current_user.contact_for(@person)
  app/views/aspect_memberships/_aspect_membership_dropdown.mobile.haml:2:in `_app_views_aspect_memberships__aspect_membership_dropdown_mobile_haml__4277730450575454658_240
435556560'
  app/helpers/aspect_global_helper.rb:27:in `aspect_membership_dropdown'
  app/views/notifications/_notification.html.haml:6:in `_app_views_notifications__notification_html_haml___774231930661711701_240470906820'
  app/controllers/notifications_controller.rb:44:in `block in index'
  app/controllers/notifications_controller.rb:43:in `each'
  app/controllers/notifications_controller.rb:43:in `index'
  lib/rack/chrome_frame.rb:39:in `call'
  lib/unicorn_killer.rb:35:in `call'
```

I didn't success to reproduce. I didn't know we try to insert the aspect dropdown in the mobile notif page.

**Comment by svbergerem** [comment]
@Flaburgan I fixed that with #5766.

**Comment by Flaburgan** [comment]
Oh, cool!
